TAF_EasyRest/TestFramework/Pages/Client/RestaurantMenuPage.cs

```diff
@@ -30,5 +21,14 @@ public CartPageComponent ClickShowCartButton()
             _showCartButton.Click();
             return new CartPageComponent(driver);
         }
+
+        public RestaurantMenuPage AddTheFistDishToTheCart()
+        {
+            var _menuItem = _menuItemSummaryCards.ElementAt(0);
```

Comment:
local variables should start with low letter without underscore, please rename the variables

Reply:
done